x-pack/plugins/data_enhanced/public/search/sessions_mgmt/lib/get_columns.tsx

```diff
@@ -113,6 +129,7 @@ export const getColumns = (
               <TableText>
                 {name}
                 {notRestorableWarning}
+                {versionIncompatibleWarning}
```

Comment:
I think we shouldn't show this warning if a session is already showing `{notRestorableWarning}` 
So we shouldn't show this, for example, for the expired sessions because we will anyway re-run then